modules/vault-agent-client.nix

```diff
@@ -73,9 +73,7 @@ let
             {{ end }}{{ end }}
           '';
 
-          command = writeShellScript "update-cert" ''
-            ${pkgs.systemd}/bin/systemctl restart certs-updated.service || true
-          '';
+          command = "${pkgs.systemd}/bin/systemctl try-reload-or-restart certs-updated.service";
```

Comment:
systemd is just full of dark corners I've never heard of and I both love and hate it at the same time.